### DISCUSSION TITLE
I try to use custom trained model to detect some pictures but I detect nothing

### DISCUSSION THREAD

**Discussion by snow-bean** [ultralytics/yolov5#7619] [Q&A] [answered]
I try to use custom trained model to detect some pictures but I detect nothing

import torch

model = torch.hub.load('/Users/hld/Downloads/yolov5-master/yolo5qstart/yolov5', 'custom', path='/Users/hld/Downloads/yolov5-master/yolo5qstart/yolov5/runs/train/exp/weights/best.pt', source='local')

im = '/Users/hld/Downloads/yolov5-master/yolo5qstart/datasets/cell40/000005.jpg'

results = model(im)

only a picture in my folder with no detect .
is there anything wrong  :model = torch.hub.load('/Users/hld/Downloads/yolov5-master/yolo5qstart/yolov5', 'custom', path='/Users/hld/Downloads/yolov5-master/yolo5qstart/yolov5/runs/train/exp/weights/best.pt', source='local')  

or else  my model is not very good ,so that could not detect ?

**Comment by glenn-jocher** [comment, marked as answer]
@snow-bean obviously test an official model on a picture of people. If that works the problem is your model being poorly trained.
